src/__tests__/detailsPage.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  fireEvent, render, screen,
+} from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { createTheme, ThemeProvider } from '@mui/material';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import Homepage from '../Pages/homePage';
+import DetailsPage from '../Pages/detailsPage';
+import countriesReducer from '../redux/Countries/countries';
+import SelectContinentReducer from '../redux/Countries/selectedContinent';
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: '#ec4c8a',
+      light: '#ef6fa1',
+      dark: '#a53560',
+    },
+    secondary: {
+      main: '#a53560',
+    },
+  },
+  typography: {
+    fontFamily: 'SegoeUI, Segoe UI',
+  },
+});
+
+const initialState = {
+  countries: [{
+    capital: ['Helsinki'],
+    continents: ['Europe'],
+    currencies: { EUR: { name: 'Euro', symbol: '€' } },
+    flag: 'https://flagcdn.com/w320/fi.png',
+    languages: { fin: 'Finnish', swe: 'Swedish' },
+    name: 'Finland',
+    nativeName: {
+      fin: { official: 'Suomen tasavalta', common: 'Suomi' },
+      swe: { official: 'Republiken Finland', common: 'Finland' },
+    },
+    population: 5530719,
+    subregion: 'Northern Europe',
+    topLevelDomain: ['.fi'],
+  }],
+  selectedContinent: 'all',
+};
+
+const rootReducer = combineReducers({
+  countries: countriesReducer,
+  selectedContinent: SelectContinentReducer,
+});
+
+const store = configureStore({
+  reducer: rootReducer,
+  preloadedState: initialState,
+});
+
+it('renders', () => {
+  render(
+    <Provider store={store}>
+      <ThemeProvider theme={theme}>
+        <BrowserRouter>
+          <Routes>
+            <Route path="/" element={<Homepage />} />
+            <Route path="/countries/:country" element={<DetailsPage />} />
+          </Routes>
+        </BrowserRouter>
+      </ThemeProvider>
+    </Provider>,
```

Comment:
- [ ] Kindly reposition the home/back button to be in the navigation bar instead of under it to match the given design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/189147442-6a3b5671-e926-4297-b8c0-6cb4f2c843cc.png) | ![image](https://user-images.githubusercontent.com/62290545/189147351-b7979520-8878-4015-910a-c5a33c27545e.png) |